--- Makefile	2016-12-25 11:55:01.000000000 +0200
+++ Makefile	2016-12-25 12:00:57.000000000 +0200
@@ -6,13 +6,6 @@
 # to build Inchworm and Chrysalis with the Intel compiler.
 #
 
-ifeq ($(TRINITY_COMPILER),intel)
- INCHWORM_CONFIGURE_FLAGS = CXX=icpc CXXFLAGS="-fast"
- CHRYSALIS_MAKE_FLAGS = COMPILER=icpc
-else
- override TRINITY_COMPILER=gnu
-endif
-
 
 all: inchworm_target chrysalis_target trinity_essentials
 	sh ./util/support_scripts/trinity_install_tests.sh
@@ -20,7 +13,7 @@
 inchworm_target:
 	@echo Using $(TRINITY_COMPILER) compiler for Inchworm and Chrysalis
 	cd Inchworm && (test -e configure || autoreconf) \
-                && sh ./configure --prefix=`pwd` $(INCHWORM_CONFIGURE_FLAGS) && $(MAKE) install
+                && sh ./configure --prefix=`pwd` CC=${PREFIX}/bin/gcc $(INCHWORM_CONFIGURE_FLAGS) CXX=${PREFIX}/bin/g++ CFLAGS="-fopenmp" CXXFLAGS="-fopenmp" && $(MAKE) install
 
 chrysalis_target:
 	cd Chrysalis && $(MAKE) UNSUPPORTED=yes $(CHRYSALIS_MAKE_FLAGS)
